### PR TITLE
Eng 611 test npm lib locally

### PR DESCRIPTION
**Pull request by ChrisSchreiber**
- added hostPath volume to helm chart to mount npm library from local filesystem 
- added skaffold profile use local-dev-values.yaml to enable and configure dev options 
- update readme

### CODE REVIEW

File: skaffold.yaml [hunk 1]
```diff
@@ -36,3 +36,20 @@ deploy:
         operators.slack.image: operator-slack
         operators.jenkins.image: operator-jenkins
         operators.toolchain.image: operator-toolchain
+profiles:
```
Comment (cplee) [inline skaffold.yaml]: I wish there was a way to DRY this up...seems very redundant

File: README.md [hunk 2]
```diff
@@ -20,6 +20,16 @@ Setup the secret for slack:
...
+## Test local changes to lead-sdm-operators-resources npm library
+
+Create `local-dev-values.yaml` file in lead-sdm-operators folder
+```
+dev:
+  enabled: true
+  path: /LOCAL/PATH/lead-sdm-operators
+```
+
+Run `skaffold dev -p dev`
```
Comment (cplee) [inline README.md]: any way to simplify this?  you can pass helm variables from the skaffold.yaml...can we remove the need for a local file that folks have to create?

Comment (ChrisSchreiber) [reply README.md]: the /LOCAL/PATH/ has to be an absolute path, my thought is that it is better that have an extra file that is in .gitignore that to edit local specific into the skaffold.yaml and then undo them before a commit.

File: skaffold.yaml (hunk 1 above)
Comment (ChrisSchreiber) [reply skaffold.yaml]: me too. if there is a way i couldn't figure it out.

Comment (ChrisSchreiber) [reply skaffold.yaml]: actually i just thought of something else to try

Comment (ChrisSchreiber) [reply skaffold.yaml]: nope that didn't work either :(

**Pull request closed without merging**